docs/data-management.rst

```diff
@@ -130,6 +130,33 @@ Browsing and rendering
 
 #. You can revert to the original settings for an Image or Dataset. For example, using the context menu for a Dataset in the tree, select ``Rendering Settings > Set Imported and Save``.
 
+
+Manage Images in Dataset/Projects
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can organize the data in the left-hand side tree by creating new Projects, Datasets and linking the Images to those new containers.
+
+#. Use the Project |image2b| and Dataset |image2c| icons above the left hand side tree or the right-click contextual menu to create new Datasets or Projects.
```

Comment:
`left-hand` (as in previous sentence)

Reply:
fixed in https://github.com/ome/omero-guide-introduction/pull/5/commits/d93c22afe0982d23c89dee2e1eb5e456a7095664